l10n_ch_base_bank/views/invoice.xml

```diff
@@ -9,11 +9,24 @@
       <field name="inherit_id" ref="account.invoice_form"/>
       <field name="arch" type="xml">
         <field name="date_due" position="after">
-          <field name="reference_type" nolabel="1"  attrs="{'readonly':[('state','!=','draft')]}"/>
-          <field name="reference" nolabel="1"  attrs="{'readonly':[('state','!=','draft')]}"/>
+          <field name="reference_type" nolabel="1"/>
```

Comment:
In the field definition in the official account module, those fields are already readonly when not in draft, cf 
https://github.com/odoo/odoo/blob/9.0/addons/account/models/account_invoice.py#L213